config/custom-environment-variables.yaml

```diff
@@ -190,6 +190,10 @@ scms:
 webhooks:
     # Obtains the SCM token for a given user. If a user does not have a valid SCM token registered with Screwdriver, it will use this user's token instead.
     username: SCM_USERNAME
+    # Ignore commits made by these users
+    ignore_commits_by:
```

Comment:
This uses _casing, noticed we are using camel casing everywhere ?

Reply:
@jithin1987 good catch! Fixed